src/uu/tail/src/tail.rs

```diff
@@ -110,14 +118,29 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         return 0;
     }
 
-    settings.follow = given_options.opt_present("f");
+    if given_options.opt_present("F") {
```

Comment:
if you are motivated, you could move this to clap (in a different PR). It would simplify the argument management

Reply:
Sure, I'll do that after this is finished.